go/protocols/capture/capture.proto

```diff
@@ -37,16 +38,23 @@ message SpecRequest {
   ];
 }
 
+// SpecResponse is the response type of the Spec RPC.
 message SpecResponse {
   // JSON schema of a complete endpoint specification.
-  string spec_schema_json = 1 [
+  string endpoint_spec_schema_json = 1 [
+    (gogoproto.casttype) = "encoding/json.RawMessage",
+    json_name = "endpointchema"
+  ];
+  // JSON schema of a complete endpoint specification.
+  string resource_spec_schema_json = 2 [
     (gogoproto.casttype) = "encoding/json.RawMessage",
     json_name = "endpointchema"
   ];
```

Comment:
Where did we land on the semantics of using this resource spec? Is that still TBD?

Typos:
- The resource spec comment mentions the endpoint.
- I'm guessing we wanted `json_name = "endpointschema"`, rather than "endpointchema".

Reply:
> Where did we land on the semantics of using this resource spec? Is that still TBD?

The `airbyte` driver does supply a schema for the resource spec, which is generated from the type it's parsed into.

There's no (current) schema validation of the resource spec. It is _pretty_ well covered by doing actual strict parsing of the resource specs as part of Validate RPCs though, for both capture and materialize. The principle motivation for adding resource specs is to make it possible for future UIs and tooling to understand and present the available configuration for a resource binding, so this is pretty forward-looking and done from a sense of wanting to carve out & codify a position on how it _should_ work.

| Typos

Doh! How about I address this in the `protocols` repo, to which these are moving ?